.idea/.gitignore

```diff
@@ -1,5 +1,5 @@
 # Default ignored files
 /workspace.xml
-.idea/
+target
 *.iml
-target/
\ No newline at end of file
+.idea
```

Comment:
Why do you need two `.gitignore` files? You have already created `${project.home}/.gitignore`

Reply:
All IDE related files added to the gitignore.